net/gce.sh

```diff
@@ -62,7 +62,7 @@ prefix=testnet-dev-${USER//[^A-Za-z0-9]/}
 additionalValidatorCount=2
 clientNodeCount=0
 blockstreamer=false
-validatorBootDiskSizeInGb=500
+validatorBootDiskSizeInGb=100
```

Comment:
why do we need a smaller boot disk size now?

Reply:
not sure why we'd need 500 GB per node. If someone really really needs it, they can set that in their own config. Testnet runs fine with 100 GB disk with 90 GB of space to spare. The only node that needs huge amount of disk is the bootstrap that builds agave, but that is hardwired to use more space.